client/e2e/pages/seasonsPage.ts

```diff
@@ -0,0 +1,165 @@
+
+  readonly seasonContinue: Locator;
+
+  readonly DashboardButton: Locator;
+  
+  readonly addEvent: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+
+    this.emailButton= page.getByText(process.env.TEST_EMAIL as string);
+
+    this.manageTicketingButton=page.getByText('Manage Ticketing').first();
+    this.dashboardSeasonsButton=page.getByRole('button', {name: 'Seasons'});
+
+    this.pageHeader = page.getByRole('heading', { name: 'Select Season' });
+
+    this.addSeasonButton = page.getByRole('button', { name: 'Add Season' });
+
+    this.seasonNameBlank = page.getByLabel('Season Name:');
+    this.seasonStartDate = page.getByLabel('Start Date:');
+    this.seasonEndDate = page.getByLabel('End Date:');
+    this.imageURL=page.getByLabel('Image URL:');
+
+    this.seasonOption1=page.getByLabel('Use Default Image');
+    this.seasonOption2=page.getByLabel('Active');
+
+    this.seasonEdit=page.locator(".flex[data-mui-internal-clone-element=true]").first();
+    this.seasonSave=page.getByRole('button', {name: 'Save'});
+    this.seasonDelete=page.locator(".flex[data-mui-internal-clone-element=true]").last();
+
+    this.seasonContinue=page.getByRole('button', { name: 'Continue' });
+
+    this.addEvent=page.getByRole('button', { name: 'Add event' });
+
+  }
+
+  async addNewSeason()
+  {
+    await this.addSeasonButton.click();
+    await this.seasonNameBlank.fill('TestName');
+    await this.seasonStartDate.fill('2020-02-02');
+    await this.seasonEndDate.fill('2020-03-02');
+    await this.imageURL.fill('https://www.hindustantimes.com/ht-img/img/2023/08/25/550x309/international_dog_day_1692974397743_1692974414085.jpg');
+    await this.seasonSave.click();
+    await this.seasonContinue.click();
+
+    await expect(this.page.locator(':text("TestName")')).toBeVisible();
+  }
+
+  async addEventToSeason() 
+  {
+    // notes:
+    // some goto methods are using loopholes to get to the page instead of the navigators
+
+    // create new event
+    const eventsPage = new EventsPage(this.page);
+    await eventsPage.goto();
+    await eventsPage.addnewevent(EventsInfo2);
+
+    // go back to seasons
+    const seasonsPage1 = new SeasonsPage(this.page);
+    await seasonsPage1.goto();
+
+    // remove from default season
+
+    // click the first season that was created previously -- assuming its there
+    await this.page.locator(':has-text("Start Date")').last().click();
+
+    await this.page.getByRole('button', { name: 'Remove Event'}).last().click();
+
+    await this.page.getByRole('button', { name: 'Continue'}).click();
+
+    // go back to seasons
+    const seasonsPage2 = new SeasonsPage(this.page);
+    await seasonsPage2.goto();
+
+    await this.page.locator(':text("TestName")').first().click();
+
+    await this.addEvent.click();
+    //await this.page.getByRole('button', { name: 'Add Event'}).click();
+    await this.page.getByRole('button', { name: 'Add to Season'}).first().click();
```

Comment:
I'd look to see if you can do something similar to lines 96-100 - use some kind of ID and filter to find what you need.  Anywhere you have to use a `.first` to find dynamic content (ie, an event or season) I'd highly recommend trying to find a more specific method to identify the locator.